renovate.json5

```
@@ -7,5 +7,12 @@
   ],
   lockFileMaintenance: {
     automerge: true
-  }
+  },
+  packageRules: [
+    {
+      matchPackageNames: ['renovate'],
+      description: '更新が多すぎるので週末にのみマージする',
+      extends: ['schedule:automergeWeekly']
```

q. PRタイトルやdescription的にはこちらに見えました（PR概要には週初めとあるのでWeeklyで合っていそうですが、、、どちらを意図していますか？）
```suggestion
      extends: ['schedule:automergeWeekends']
```

---

あー、正直どっちでもいいですが、Weekendsだと土日になるのでそれよりかはWeekly(=earlyMondays)の方が良いかなと思っております。PRのタイトルを修正しておきます。

---

承知です。descriptionも同じ修正をお願いします:pray:

---

ref: 35d5e3f50d3b5b51fcacc0a98497905c8c399f9a

---

すみません。すごいIMOなんですが、パット見たときにいつの何時かわからないので、Cronと同様の記法するのって微妙ですかね？
`* 10 * * 5` みたいな。

Weekendsって言われて金曜日なのか日曜日なのかぱっと分からなかった人:innocent:
https://docs.renovatebot.com/presets-schedule/

---

https://crontab.guru/
どうぞ

---

あ、 `automergeWeekends` って言われても分からなかったので `* 10 * * 5` にしませんか？の意でした。

---

できれば標準で用意されているpresetを使いたいのでリンクを貼っておきます

---

ま、本当の目的はマージが多すぎるのを抑止したいって話なので、できれば別の方法があるのならその方がいいのでしょうけど

---

公式ドキュメントに容易にたどり着けたので正直どっちでもよいです（どうせcronでも何番目がどの意味だっけ？をググるだろうから初見でわからないことに変わりない民ｗ）
お好みで何卒

- `renovate schedule:automergeWeekly` でググる
- https://docs.renovatebot.com/presets-schedule/#scheduleautomergeweekly に到達
- `schedule:automergeEarlyMondays` をページ内検索
- https://docs.renovatebot.com/presets-schedule/#scheduleautomergeearlymondays に到達
- `Schedule automerge on Monday mornings (before 4 AM).` とわかる

---

> できれば標準で用意されているpresetを使いたい

:yoshi:

そうですねｗドキュメント見たら分かる。は本当にそうなのでどっちでも大丈夫ではありますｗ